src/index.html

```diff
@@ -11,12 +11,107 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header_logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_item">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li>
```